src/apps/langchain/pyproject.toml

```diff
@@ -29,9 +29,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.9,<3.13"
 trulens-core = { version = "^1.0.0" }
-langchain = "^0.2.10,!=0.2.12"
-langchain-core = "^0.2.24,!=0.2.27"
-typing-inspect = "^0.8"
```

Comment:
@sfc-gh-pmardziel Do we still depend on this? I couldn't find where this is used.

Reply:
I don't see it either. Lets remove and see if anything breaks.